style.css

```diff
@@ -121,3 +120,96 @@
   display: flex;
   gap: 15px;
 }
+
+.icons a {
+  color: #fff;
+}
+
+.header {
+  display: grid;
+  grid-template-columns: auto;
+  justify-content: center;
+}
+
+.recent {
+  color: #9490ff;
+  font-weight: 500;
+  font-size: 40px;
+  line-height: 52px;
+  width: 327px;
+  padding-left: 25px;
+}
+
+.lang-sec {
+  background-image: url(./mobile.png);
+  background-repeat: no-repeat;
+  display: flex;
+  background-position: 0 0;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 25px;
+}
+
+.header hr {
+  width: 323px;
+  border: 1px solid #646369;
+  margin: 44px 24px;
+}
+
+.section1 {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.text {
+  font-size: 20px;
+  color: #dcdce8;
+  width: 327px;
+  left: 0;
+  font-weight: 400;
+  top: 129px;
+}
+
+.post {
+  color: #fff;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 32px;
+}
+
+.lang {
+  display: flex;
```

Comment:
### OPTIONAL
- [ ] For DRY CSS you should combine duplicate properties. Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.Flex is just an example you can do it to duplicate properties.



- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and css best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)